orionpi.sh

```diff
@@ -178,7 +178,7 @@ fi
 printf "\n\r[\033[0;32m✓\033[0m]\033[0;32m%s\033[0m\n" "  Running OrionPi on Raspberry Pi!"
 printf "\r[i]""  Press \033[0;31mCtrl+C\033[0m to disconnect.\n\n"
 if [ "$release" = true ]; then
-    sshpass -p "$password" ssh $user@$ip 'flutter-pi --release --pixelformat=RGB565 /home/pi/orionpi/flutter_assets'
+    sshpass -p "$password" ssh $user@$ip 'flutter-pi --release --pixelformat=RGB565 /home/pi/orion'
 else
-    sshpass -p "$password" ssh $user@$ip 'flutter-pi --pixelformat=RGB565 /home/pi/orionpi/flutter_assets'
+    sshpass -p "$password" ssh $user@$ip 'flutter-pi --pixelformat=RGB565 /home/pi/orion'
```

Comment:
Should use `/home/$user/orion` in this line and the above

Reply:
Resolved by [d51d0b3](https://github.com/TheContrappostoShop/Orion/pull/14/commits/d51d0b3020a30981e5f31d693921ff2f66f1146d)